### PR TITLE
Cypress: update to 5.2.0

### PR DESCRIPTION
**Pull request by alejandra-martinez**
- [x] Identified the issue which this PR solves.
- [x] Read the [**CONTRIBUTING**](https://github.com/target/goalert/blob/master/CONTRIBUTING.md) document.
- [x] Code builds clean without any errors or warnings.
- [x] Added appropriate tests for any new functionality.
- [x] All new and existing tests passed.
- [x] Added comments in the code, where necessary.
- [x] Ran `make check` to catch common errors. Fixed any that came up.

**Description:**
This PR updates Cypress to 5.2.0, which adds built-in test retries.

**Which issue(s) this PR fixes:**
Brings Cypress up to date with current version.

### CODE REVIEW

File: web/src/cypress/support/index.ts [hunk 1]
```diff
@@ -36,9 +34,9 @@ import './sql'
 import './form'
 import './dialog'
 
-Cypress.env('RETRIES', 2)
+Cypress.env('retries', 1)
```
Comment (mastercactapus) [inline web/src/cypress/support/index.ts:37]: Was it intentional to change retries to 1, down from 2?

File: web/src/cypress/tsconfig.json [hunk 2]
```diff
@@ -1,5 +1,8 @@
 {
   "extends": "../app/tsconfig",
   "include": ["**/*"],
-  "types": ["./types/*", "../schema.d.ts"]
+  "types": ["./types/*", "../schema.d.ts"],
+  "compilerOptions": {
+    "esModuleInterop": true
+  }
```
Comment (mastercactapus) [inline web/src/cypress/tsconfig.json:7]: This is already set in `../app/tsconfig` so we can remove it here

**Review by mastercactapus — CHANGES_REQUESTED** [review]

File: web/src/cypress/support/index.ts (hunk 1 above)
Comment (alejandra-martinez) [reply web/src/cypress/support/index.ts:37]: yep! thats what the migration doc says. https://docs.cypress.io/guides/references/migration-guide.html#Configure-test-retries-in-the-configuration-file

Comment (mastercactapus) [reply web/src/cypress/support/index.ts:37]: It looks like their example is just misleading, we want 3 attempts and we'd still need to set retries to 2 in order to get that behavior -- however it looks like we can delete this line and instead add `"retries": 2` to `cypress.json` based on that doc you linked.

Comment (alejandra-martinez) [reply web/src/cypress/support/index.ts]: gotcha, ok, all fixed!

**Review by mastercactapus — APPROVED** [review]

**Review by m17ch — APPROVED** [review]
LGTM 👍

**Pull request merged**